flask_api/app.py

```diff
@@ -55,7 +55,7 @@ def make_response(self, rv):
             headers, status_or_headers = status_or_headers, None
 
         if not isinstance(rv, self.response_class):
-            if isinstance(rv, (text_type, bytes, bytearray, list, dict)):
+            if isinstance(rv, self.response_class.api_return_types):
```

Comment:
I think this might have resulted from incorrectly resolving a merge conflict. Investigating...